include/tsutil/Bravo.h

```diff
@@ -251,7 +251,7 @@ template <typename T = std::shared_mutex, size_t SLOT_SIZE = 256, int SLOWDOWN_G
   void
   lock_shared(Token &token)
   {
-    ink_assert(SLOT_SIZE >= DenseThreadId::num_possible_values());
+    debug_assert(SLOT_SIZE >= DenseThreadId::num_possible_values());
```

Comment:
I haven't seen `debug_assert` before, could you explain this change?

Reply:
This is a following change of f600c164c910e192790c559a2a5f903316da8f60, that tried to make `tsutil` independent from `tscore`.

https://github.com/apache/trafficserver/commit/f600c164c910e192790c559a2a5f903316da8f60#diff-3c5cec5927ea8c7c2e772096d358a3724d36e9884cb05395d7f511e52647b92cL40-L43